src/System.Linq.Expressions/tests/SequenceTests/SequenceTests.cs

```diff
@@ -2960,29 +2929,6 @@ public static void InvokeQuotedLambda(bool useInterpreter)
             Assert.Equal(6, d());
         }
 
-        [Theory(Skip = "870811")]
```

Comment:
The only one that i could find by this ID is about .Net native being unable to handle some case with a lambda inside an iterator because it would result in needing to access iterator display class dynamically. In some cases the dynamic access to classes needs to be opted-in at AOT compile time which would be a problem when dealing with compiler generated type.

It looks like this case is a simplified version of that scenario - we are trying to compile and execute a lambda dynamically, possibly to see if we still can get variables captured by statically compiled lambda from the closure class.

Perhaps it is better to enable this test as an isolated change, so that it could be reverted in a case if the issue with AOT is still there. I am not sure if CI tests can ensure that.